js/libs/builder.js

```diff
@@ -83,7 +84,10 @@ var app = new Vue({
       toChangeTemplate: false,
       permissionToChange: false,
       newTemplate: '',
-      redirect: formSettings.redirect
+      redirect: formSettings.redirect,
+      toggleTemplatedEmail: formSettings.onSubmit.indexOf("templatedEmail") > -1 ? true : false,
```

Comment:
single quotes everywhere please :)
also: no need to do `? true : false`, just write the operation (`foo > -1`)